examples/lynxCapital/tests/test_providerlab.py

```
@@ -1877,11 +1877,169 @@ def test_api_key_pair_distinct_cases():
+                  json={"documentId": started["documentId"]}).json()["data"]
+    assert done["status"] in ("extracted", "needs_review")
+    assert done["extractionId"].startswith("ext_")
+    assert done["corrections"] == []
+    assert isinstance(done["fullText"], str) and len(done["fullText"]) > 80
+    assert done["fullText"].count("\n") > 5
+    assert isinstance(done["pages"], list) and len(done["pages"]) >= 1
+    page = done["pages"][0]
+    for k in ("pageNumber", "width", "height", "unit", "angle", "dpi", "detectedLanguages", "text"):
+        assert k in page
+    assert page["detectedLanguages"] and "language" in page["detectedLanguages"][0]
+    for name, field in done["fields"].items():
```

Ruff (0.15.17)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)